rules/unauthenticated-service/test/service/expected.json

```diff
@@ -0,0 +1,83 @@
+{
+    "alertMessage": "Unauthenticated service operator exposes operator",
+    "alertObject": {
+      "k8sApiObjects": [
+        {
+          "apiVersion": "v1",
+          "kind": "Pod",
+          "metadata": {
+            "name": "operator",
+            "labels": {
+              "app": "operator"
+            }
+          }
+        }
+      ]
+    },
+    "alertScore": 7,
+    "deletePaths": null,
+    "failedPaths": ["spec"],
+    "fixPaths": [],
+    "packagename": "armo_builtins",
+    "relatedObjects": [
+      {
+        "deletePaths": null,
+        "failedPaths": null,
+        "fixPaths": null,
+        "object": {
+          "apiVersion": "v1",
+          "kind": "Service",
+          "metadata": {
+            "name": "operator",
+            "namespace": "kubescape"
+          },
+          "spec": {
+            "ports": [
+              {
+                "port": 4002,
+                "protocol": "TCP",
+                "targetPort": 8080
+              }
+            ],
+            "selector": {
+              "app": "operator"
+            }
+          }
+        },
+        "reviewPaths": null
+      },
+      {
+        "deletePaths": null,
+        "failedPaths": null,
+        "fixPaths": null,
+        "object": {
+          "apiVersion": "kubescape.io/v1",
+          "kind": "ServiceScanResult",
+          "metadata": {
+            "creationTimestamp": "2024-07-03T04:40:17Z",
+            "generation": 4,
+            "name": "operator",
+            "namespace": "kubescape",
+            "resourceVersion": "2772",
+            "uid": "24dc622d-ee78-40c2-8654-2a5604715f95"
+          },
+          "spec": {
+            "clusterIP": "10.103.207.220",
+            "ports": [
+              {
+                "applicationLayer": "",
+                "authenticated": false,
+                "port": 4002,
+                "presentationLayer": "http",
+                "protocol": "TCP",
+                "sessionLayer": "tcp"
+              }
+            ]
+          }
+        },
+        "reviewPaths": null
+      }
+    ],
+    "reviewPaths": ["spec"],
+    "ruleStatus": ""
+}
```

Comment:
We should also add a test for "passing".

Reply:
I have added yaml's that doesn't fail.